docker-compose-web-modeler-beta.yaml

```
@@ -0,0 +1,165 @@
+# Docker compose file for Web Modeler Self-Managed Beta Release [*]. This file is not intended to be used stand-alone.
+# Use it in combination with docker-compose.yml:  
+#
+#    docker-compose -f docker-compose.yaml -f docker-compose-web-modeler-beta.yaml up -d
+#
+# [*] Web Modeler Self-Managed is currently offered as a beta release [1] with limited availability for enterprise customers only.
+# It is not recommended for production use, and there is no maintenance service guaranteed. Special terms & conditions [2] apply. 
+# However, we encourage you to provide feedback via your designated support channel or the Camunda Forum [3].
+# 
+# Links
+# [1] - https://docs.camunda.io/docs/next/reference/early-access#beta
+# [2] - https://camunda.com/legal/terms/camunda-platform/camunda-platform-8-self-managed/
+# [3] - https://forum.camunda.io/
+
+# Notes: this file is using mailhog to simulate a Mail server
+
+# TODO 
+# - swap latest tag with release version
+
+version: "2.4"
+
+services:
+
+  modeler-db:
+    container_name: modeler-db
+    image: postgres:14.2-alpine
+    ports:
+      - "5433:5432"
+    healthcheck:
+      test: pg_isready -d modeler-db -U modeler-db-user
+      interval: 5s
+      timeout: 15s
+      retries: 30
+    environment:
+      POSTGRES_DB: modeler-db
+      POSTGRES_USER: modeler-db-user
+      POSTGRES_PASSWORD: modeler-db-password
+    networks:
+      - modeler      
+
+  modeler-websockets:
+    container_name: websockets
+    image: registry.camunda.cloud/cawemo-saas/cawemo-websockets:latest
+    ports:
+      - "8060:8060"
+    healthcheck:
+      test: /usr/bin/nc -v localhost 8060
+      interval: 30s
+    environment:
+      APP_NAME: "Web Modeler Self-Managed WebSockets"
+      APP_DEBUG: "true"
+      PUSHER_APP_ID: modeler-app
+      PUSHER_APP_KEY: modeler-app-key 
+      PUSHER_APP_SECRET: modeler-app-secret
+      PUSHER_APP_CLUSTER: modeler
+    networks:
+      - modeler      
+
+  mailhog:
+    # If you want to use your own SMTP server, you can remove this container
+    # and configure RESTAPI_MAIL_HOST, RESTAPI_MAIL_PORT and RESTAPI_MAIL_ENABLE_TLS
+    # accordingly
```

Real SMTP servers probably require authentication:

```suggestion
    # If you want to use your own SMTP server, you can remove this container
    # and configure RESTAPI_MAIL_HOST, RESTAPI_MAIL_PORT, RESTAPI_MAIL_USER, RESTAPI_MAIL_PASSWORD, and RESTAPI_MAIL_ENABLE_TLS
    # accordingly
```